variables.tf

```diff
@@ -15,6 +15,10 @@ variable "engine_version" {
   type        = string
   description = "The engine version to use."
 }
+variable "family" {
+  type        = string
+  description = "engine family to assign to the parameter group"
+}
```

Comment:
There wasn't a good way to deduce what the database/engine family because the version might have more than 2 digits. I guess we could try to construct it, but it the constructed family might not exist. Thoughts?